Tests/Unit/EmogrifierTest.php

```diff
@@ -1848,7 +1891,8 @@ public function dataUrisAreConserved()
     {
         $html = $this->html5DocumentType . '<html></html>';
         $this->subject->setHtml($html);
-        $styleRule = 'background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAIAAAAC64paAAABUk' .
+        $styleRule = 'background-image: url(data:image/png;charset=utf-8;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUC' .
```

Comment:
If I understand https://developer.mozilla.org/en-US/docs/Web/HTTP/data_URIs correctly, the following cases should be valid:

data:image/png,iVBORw0KG...
data:image/png;charset=utf-8,iVBORw0KG...
data:image/png;base64,iVBORw0KG...
data:image/png;charset=utf-8;base64,iVBORw0KG...

It'd like to have all these covered by tests. I propose having a data provider for the part between "data:image/png" and the base64-encoded data.